app/code/local/Magmodules/Webwinkelconnect/Helper/Data.php

```diff
@@ -195,17 +202,18 @@ public function syncProductReview(array $product_review): array {
             $review->setCreatedAt($product_review['product_review']['created']);
             $review->save();
 
+            $review_edit_time = microtime(true) - $review_edit_time_start;
+
             $connection->commit();
```

Comment:
Best move the `commit` right before the `return`, so the transaction also includes the logging and it's not possible to get a review without a log message.